### PR TITLE
Add csv renderer

### PR DESCRIPTION
**Pull request by PascalEgn**
ref: https://github.com/cern-sis/issues-scoap3/issues/181

### CODE REVIEW

File: scoap3/articles/api/views.py [hunk 1]
```diff
@@ -54,10 +58,24 @@ class ArticleDocumentView(DocumentViewSet):
...
 
     search_fields = ("title", "id")
     permission_classes = [IsAuthenticatedOrReadOnly]
 
+    def get_renderer_context(self):
+        renderer_context = super().get_renderer_context()
+        requested_renderer_format = self.request.accepted_media_type
+        if "text/csv" in requested_renderer_format:
+            pagination.PageNumberPagination.page_size = 100
+        return renderer_context
```
Comment (drjova) [inline scoap3/articles/api/views.py]: I think we can use `max_page_size` as here https://www.django-rest-framework.org/api-guide/pagination/#modifying-the-pagination-style

File: scoap3/articles/api/views.py [hunk 2]
```diff
@@ -54,10 +58,24 @@ class ArticleDocumentView(DocumentViewSet):
...
+
+    def get_serializer_class(self):
+        requested_renderer_format = self.request.accepted_media_type
+        if "text/csv" in requested_renderer_format:
+            return SearchCSVSerializer
+        return ArticleDocumentSerializer
```
Comment (drjova) [inline scoap3/articles/api/views.py:69]: do we need to hardcode the serialisers in the viewset? Did we check of a config var or something similar?

File: scoap3/articles/api/views.py (hunk 1 above)
Comment (PascalEgn) [reply scoap3/articles/api/views.py]: I don't see the point of changing it until we have further details.
Also if we want `max_page_size` of `100` we can also remove the pagination line here and rely on the global pagination: https://github.com/SCOAP3/scoap3/blob/main/scoap3/utils/pagination.py

Comment (drjova) [reply scoap3/articles/api/views.py]: Yes, let's rely in the global one and not hardcode the value 👍

**Review by drjova — APPROVED** [review]

**Pull request merged**